data/global listeners.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 by RisingLeaf(https://github.com/RisingLeaf)
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Unlock Cloaking Key"
+	invisible
+	landing
+	to offer
+		and
+			not "global: found cloaking"
```

Comment:
1. The `and` is unnecessary.
2. This `not` condition is basically unnecessary because `set` only ever sets the value of a condition to 1. It doesn't matter if the player starts a new pilot, gets a cloak, and has this mission offer again, because the condition will still only be 1.

Reply:
removed the and, would prefer to have the not, cause we do not need this to clutter the save files

Reply:
It's one extra line to say that this mission got offered. I'd hardly consider that clutter. We'd also be able to use the offering of this mission to check if a particular pilot has used a cloaking device before.

Reply:
in that case I will remove it

Reply:
removed